src/type/schema.js

```diff
@@ -159,6 +160,8 @@ export class GraphQLSchema {
             this._implementations[iface.name] = [type];
           }
         });
+      } else if (isAbstractType(type) && !this._implementations[type.name]) {
+        this._implementations[type.name] = [];
```

Comment:
guarantees that `getPossibleTypes` returns an array, as it should per its flow type.